lib/sqlalchemy/sql/selectable.py

```diff
@@ -2755,7 +2780,7 @@ def cte(
             :meth:`_expression.HasCTE.cte`.
 
         """
```

Comment:
**mike bayer** (zzzeek) wrote:

remove

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4325